glean-core/android/src/main/java/mozilla/telemetry/glean/private/CounterMetricType.kt

```diff
@@ -56,14 +57,7 @@ class CounterMetricType internal constructor(
         }
     }
 
-    private fun shouldRecord(): Boolean {
-        // Don't record metrics if we aren't initialized
-        if (!Glean.isInitialized()) {
-            return false
-        }
-
-        return LibGleanFFI.INSTANCE.glean_counter_should_record(Glean.handle, this.handle).toBoolean()
```

Comment:
Can you please file a follow-up bug for removing the `glean_*_should_record` functions from the FFI layer? I don't think we need them anymore.

Reply:
Bug filed [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1569603)